pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -79,7 +73,8 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 		//   https://github.com/opencontainers/runtime-spec/blob/ad53dcdc39f1f7f7472b10aa0a45648fe4865496/config-windows.md#cpu
 		//   If both CpuWeight and CpuMaximum are set - ContainerD catches this invalid case and returns an error instead.
 
-		cpuMaximum := 10000 * cpuLimit.MilliValue() / int64(runtime.NumCPU()) / 1000
+		numCPU := int64(winstats.ProcessorCount())
```

Comment:
One suggestion would be to inline this as it was done before instead of introducing a single use variable.

Reply:
for testing and clarity reasons I extracted the calculation into an own function but did inline the `numCPU` again. LMK if that's fine.

Reply:
That's even better @mweibel